homeassistant/components/deconz/services.py

```diff
@@ -59,14 +59,28 @@ async def async_call_deconz_service(service_call):
         service = service_call.service
         service_data = service_call.data
 
+        gateway = get_master_gateway(hass)
+        if CONF_BRIDGE_ID in service_data:
+            found_gateway = False
+            bridge_id = normalize_bridge_id(service_data[CONF_BRIDGE_ID])
+
+            for possible_gateway in hass.data[DOMAIN].values():
+                if possible_gateway.bridgeid == bridge_id:
+                    gateway = possible_gateway
+                    found_gateway = True
```

Comment:
Should we break this loop when found?

```suggestion
            for possible_gateway in hass.data[DOMAIN].values():
                if possible_gateway.bridgeid == bridge_id:
                    gateway = possible_gateway
                    found_gateway = True
                    break
```